src/controller/CHIPDeviceController.h

```diff
@@ -291,13 +290,14 @@ class DLL_EXPORT DeviceController : public Messaging::ExchangeDelegate,
     CHIP_ERROR ServiceEvents();
 
     /**
-     * @brief Get the Fabric ID assigned to the device.
-     *
-     * @param[out] fabricId   Fabric ID of the device.
-     *
-     * @return CHIP_ERROR CHIP_NO_ERROR on success, or corresponding error code.
+     * @brief Get the Compressed Fabric ID assigned to the device.
      */
-    CHIP_ERROR GetFabricId(uint64_t & fabricId);
+    uint64_t GetCompressedFabricId() const { return mLocalId.GetCompressedFabricId(); }
+
+    /**
+     * @brief Get the raw Fabric ID assigned to the device.
+     */
+    uint64_t GetRawFabricId() const { return mRawFabricId; }
```

Comment:
I think the 'raw fabric ID' terminology is well-meaning and an attempt for symmetry with 'compressed fabric ID'.

However, all of this is confusing enough and the spec has already defined terms for these things:

* fully-qualified fabric reference: the <root CA public key, fabric ID> tuple
* fabric ID: self-explanatory
* compressed fabric ID/identifier: the 64-bit hash of <root CA public key, fabric ID> we're now using in this PR

Given this, can we just stick to `FabricId` instead of `RawFabricId`?  This is the terminology the spec uses and that should be our guide.